### PR TITLE
[ruby] reduce INFO log for server CQ pluck registration to DEBUG

### PR DESCRIPTION
**Pull request by apolcyn**
Currently this is still causing log spam for ruby.

Though this is in core, I believe this code path is specific to ruby.

### CODE REVIEW

**Review by ctiller — APPROVED** [review]
please wait until after flake repo freeze to merge

**Comment by chadlwilson** [comment]
Thanks for looking at this. Sorry if this is a slightly off topic question, but this is the last such info log that is visible for us in the log spam.

My question is when this one is gone, will the `WARNING: All log messages before absl::InitializeLog() is called are written to STDERR` log that precedes also disappear? I inferred that this WARNING is only written when something is output to STDOUT/STDERR so hoping this one goes away too.

**Comment by apolcyn** [comment]
@chadlwilson yes that warning will go away as you described

**Pull request closed without merging**
